nibabel/nicom/tests/data/ascconv_sample.txt

```diff
@@ -0,0 +1,919 @@
+asCoilSelectMeas[0].sCOILPLUGS.aulPlugId[1] = 0xee
+asCoilSelectMeas[0].sCOILPLUGS.aulPlugId[2] = 0xee
+asCoilSelectMeas[0].sCOILPLUGS.aulPlugId[3] = 0xad
+asCoilSelectMeas[0].sCOILPLUGS.aulPlugId[4] = 0xee
+asCoilSelectMeas[0].sCOILPLUGS.aulPlugId[5] = 0xee
+asCoilSelectMeas[0].sCOILPLUGS.aulPlugId[6] = 0x5d
+asCoilSelectMeas[0].sCOILPLUGS.aulPlugId[7] = 0xb1
+asCoilSelectMeas[0].sCOILPLUGS.aulPlugId[8] = 0xee
+asCoilSelectMeas[0].sCOILPLUGS.aulPlugId[9] = 0xb2
+asCoilSelectMeas[0].sCOILPLUGS.aulPlugId[10] = 0xee
+asCoilSelectMeas[0].sCOILPLUGS.auiNmbrOfNibbles[0] = 0x2
+asCoilSelectMeas[0].sCOILPLUGS.auiNmbrOfNibbles[1] = 0x2
+asCoilSelectMeas[0].sCOILPLUGS.auiNmbrOfNibbles[2] = 0x2
+asCoilSelectMeas[0].sCOILPLUGS.auiNmbrOfNibbles[3] = 0x2
+asCoilSelectMeas[0].sCOILPLUGS.auiNmbrOfNibbles[4] = 0x2
+asCoilSelectMeas[0].sCOILPLUGS.auiNmbrOfNibbles[5] = 0x2
+asCoilSelectMeas[0].sCOILPLUGS.auiNmbrOfNibbles[6] = 0x2
+asCoilSelectMeas[0].sCOILPLUGS.auiNmbrOfNibbles[7] = 0x2
+asCoilSelectMeas[0].sCOILPLUGS.auiNmbrOfNibbles[8] = 0x2
+asCoilSelectMeas[0].sCOILPLUGS.auiNmbrOfNibbles[9] = 0x2
+asCoilSelectMeas[0].sCOILPLUGS.auiNmbrOfNibbles[10] = 0x2
+asCoilSelectMeas[0].aFFT_SCALE[0].flFactor = 3.77259
+asCoilSelectMeas[0].aFFT_SCALE[0].bValid = 1
+asCoilSelectMeas[0].aFFT_SCALE[0].lRxChannel = 1
+asCoilSelectMeas[0].aFFT_SCALE[1].flFactor = 3.83164
+asCoilSelectMeas[0].aFFT_SCALE[1].bValid = 1
+asCoilSelectMeas[0].aFFT_SCALE[1].lRxChannel = 2
+asCoilSelectMeas[0].aFFT_SCALE[2].flFactor = 3.7338
+asCoilSelectMeas[0].aFFT_SCALE[2].bValid = 1
+asCoilSelectMeas[0].aFFT_SCALE[2].lRxChannel = 3
+asCoilSelectMeas[0].aFFT_SCALE[3].flFactor = 4.08449
+asCoilSelectMeas[0].aFFT_SCALE[3].bValid = 1
+asCoilSelectMeas[0].aFFT_SCALE[3].lRxChannel = 4
+asCoilSelectMeas[0].aFFT_SCALE[4].flFactor = 3.82172
+asCoilSelectMeas[0].aFFT_SCALE[4].bValid = 1
+asCoilSelectMeas[0].aFFT_SCALE[4].lRxChannel = 5
+asCoilSelectMeas[0].aFFT_SCALE[5].flFactor = 3.86816
+asCoilSelectMeas[0].aFFT_SCALE[5].bValid = 1
+asCoilSelectMeas[0].aFFT_SCALE[5].lRxChannel = 6
+asCoilSelectMeas[0].aFFT_SCALE[6].flFactor = 4.48252
+asCoilSelectMeas[0].aFFT_SCALE[6].bValid = 1
+asCoilSelectMeas[0].aFFT_SCALE[6].lRxChannel = 7
+asCoilSelectMeas[0].aFFT_SCALE[7].flFactor = 4.39406
+asCoilSelectMeas[0].aFFT_SCALE[7].bValid = 1
+asCoilSelectMeas[0].aFFT_SCALE[7].lRxChannel = 8
+asCoilSelectMeas[0].aFFT_SCALE[8].flFactor = 4.50498
+asCoilSelectMeas[0].aFFT_SCALE[8].bValid = 1
+asCoilSelectMeas[0].aFFT_SCALE[8].lRxChannel = 9
+asCoilSelectMeas[0].aFFT_SCALE[9].flFactor = 4.57011
+asCoilSelectMeas[0].aFFT_SCALE[9].bValid = 1
+asCoilSelectMeas[0].aFFT_SCALE[9].lRxChannel = 10
+asCoilSelectMeas[0].aFFT_SCALE[10].flFactor = 4.6211
+asCoilSelectMeas[0].aFFT_SCALE[10].bValid = 1
+asCoilSelectMeas[0].aFFT_SCALE[10].lRxChannel = 11
+asCoilSelectMeas[0].aFFT_SCALE[11].flFactor = 4.69845
+asCoilSelectMeas[0].aFFT_SCALE[11].bValid = 1
+asCoilSelectMeas[0].aFFT_SCALE[11].lRxChannel = 12
```

Comment:
Currently we don't parse these as lists, instead the key ends up being 'asCoilSelectMeas[0].aFFT_SCALE[6].lRxChannel'. 

It would be nice to convert these to lists, but then we need to deal with some potential weirdness (like it seems some lists are using 0-based indexing and others are using 1-based indexing).